iocage_lib/utils.py

```diff
@@ -0,0 +1,26 @@
+import six
+import sys
+
+from ctypes import CDLL
+from ctypes.util import find_library
+
+
+def load_ctypes_library(name, signatures):
+    library_name = find_library(name)
+    if not library_name:
+        raise ImportError('No library named %s' % name)
+    lib = CDLL(library_name, use_errno=True)
+    # Add function signatures
+    for func_name, signature in signatures.items():
+        function = getattr(lib, func_name, None)
```

Comment:
can you explain how this loop affects the returned `lib`?

Reply:
https://docs.python.org/3/library/ctypes.html#specifying-the-required-argument-types-function-prototypes